<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QStackedWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">StackedWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="page">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Login name / E-mail:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="login">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="password">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="rememberPassword">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Remember password</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loginButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/images/images/login.png</normaloff>:/images/images/login.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>469</width>
        <height>23</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>465</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Logging in...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="progressLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>:/images/images/loading.gif</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/images/images/clear.png</normaloff>:/images/images/clear.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_2">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>355</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="username">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string notr="true">You are logged in as %USERNAME%</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="logoutButton">
      <property name="text">
       <string>Logout</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/images/images/login.png</normaloff>:/images/images/login.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>loginButton</sender>
   <signal>released()</signal>
   <receiver>LoginWidget</receiver>
   <slot>logIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>683</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logoutButton</sender>
   <signal>released()</signal>
   <receiver>LoginWidget</receiver>
   <slot>logOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>released()</signal>
   <receiver>LoginWidget</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>login</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>683</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>password</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>683</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rememberPassword</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LoginWidget</receiver>
   <slot>rememberChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>logIn()</slot>
  <slot>logOut()</slot>
  <slot>cancel()</slot>
  <slot>rememberChange()</slot>
 </slots>
</ui>
